UI/src/view/MassTransitStopLayer.js

```diff
@@ -354,14 +354,15 @@ window.MassTransitStopLayer = function(map, roadCollection, mapOverlay, assetGro
   };
 
   var createNewAsset = function(lonlat, placement) {
+    var directions_decode = {BothDirections: 2, TowardsDigitizing: 2, AgainstDigitizing: 3};
```

Comment:
I agree with your sugestion, however it was done this way because the specification says the following -> " If road link traffic direction is "both directions", the default side code value for bus stop can be "towards digitizing". 
Please, note that the backend isn't prepared to work with value 1 (BothDirections).